### PR TITLE
Add query syntax to the `test-client`

### PR DESCRIPTION
**Pull request by hermitkim1**
Add query syntax to the `test-client`

- Press the 'ENTER' key to continue
- Ask to try to deploy a single Kubernetes cluster across multi-clouds
- Add MCIS clean-up
- Ask to clean up MCIS

### CODE REVIEW

**Comment by hermitkim1** [comment]
It's getting to be a demo-client 😅

File: poc-cb-net/cmd/test-client/test-client.go [hunk 1]
```diff
@@ -40,6 +42,54 @@ func init() {
 	fmt.Println("End......... init() of test-client.go")
 }
 
+func pressEnterKeyToContinue(isOn bool) {
+	if isOn {
+		fmt.Println("##### Press the 'ENTER' key to continue...")
+		var input string
+		fmt.Scanln(&input)
```
Comment (seokho-son) [inline poc-cb-net/cmd/test-client/test-client.go:49]: if don't want to continue ... exit ?

File: poc-cb-net/cmd/test-client/test-client.go [hunk 2]
```diff
@@ -40,6 +42,54 @@ func init() {
...
+	}
+}
+
+func insertK8sCommands(isOn bool) string {
+	if isOn {
+		fmt.Println("\n\n[Usage] You can enter a shortcut key below or insert your 'kubectl' commands")
```
Comment (seokho-son) [inline poc-cb-net/cmd/test-client/test-client.go:55]:
```suggestion
		fmt.Println("\n\n[Usage] Select a shortcut command below or Enter manual 'kubectl' commands")
```
? ^^

File: poc-cb-net/cmd/test-client/test-client.go [hunk 3]
```diff
@@ -101,67 +151,90 @@ func deployKubernetesCluster(nsID string, mcisID string, vmID string) {
...
+			defer wg.Done()
+			resp, err = client.R().
+				SetHeader("Content-Type", "application/json").
+				SetHeader("Accept", "application/json").
+				SetPathParams(map[string]string{
+					"nsId":   nsID,
+					"mcisId": mcisID,
+				}).
+				SetBody(body).
+				Post("http://localhost:1323/tumblebug/ns/{nsId}/cmd/mcis/{mcisId}")
```
Comment (seokho-son) [inline poc-cb-net/cmd/test-client/test-client.go:219]: Managing TB endpoint in a variable is even better :) 
endpointTB := "http://localhost:1323" ?

File: poc-cb-net/cmd/test-client/test-client.go [hunk 4]
```diff
@@ -472,8 +592,60 @@ func main() {
 	time.Sleep(5 * time.Second)
 
 	// Special stage ;)
-	// Deploy a single Kubernetes cluster across multi-clouds (i.e., on MCIS)
-	deployKubernetesCluster(nsID, mcisID, retVMID.String())
+	char := askYesOrNoQuestion(isOn, "Do you want to try to deploy a single Kubernetes cluster across multi-clouds? (y/n)")
```
Comment (seokho-son) [inline poc-cb-net/cmd/test-client/test-client.go:595]:
```suggestion
	char := askYesOrNoQuestion(isOn, "Do you want to deploy a single Kubernetes cluster across multi-clouds? (y/n)")
```
for simplicity

**Review by seokho-son — COMMENTED** [review]
minor comments ;)

File: poc-cb-net/cmd/test-client/test-client.go (hunk 1 above)
Comment (hermitkim1) [reply poc-cb-net/cmd/test-client/test-client.go:49]: There is no way out currently 😅

**Comment by hermitkim1** [comment]
@seokho-son, thank you for your comments ^^

**Pull request merged**